.github/workflows/deploy_tre.yml

```diff
@@ -51,6 +46,7 @@ jobs:
       - name: Deploy TRE
         shell: bash
         env:
+          TF_IN_AUTOMATION: 1
```

Comment:
Should this be part of this PR?

Reply:
Yes, if you can add these as secrets in your build,       -e ARM_CLIENT_ID \
      -e ARM_CLIENT_SECRET \
      -e ARM_TENANT_ID \
      -e ARM_SUBSCRIPTION_ID \

Reply:
What's this got to do with the docker compose file? Thought this was a separate issue regarding amending the deployment scripts?

Reply:
Hey @marrobi. I had a chat with @ross-p-smith and this change isn't part of the move away from Compose so he'll remove this file from the PR.

Reply:
I was trying to be nice, but I will fix up GH Actions as a separate task